src/modules/mc_pos_control/MulticopterPositionControl.cpp

```diff
@@ -190,6 +190,28 @@ void MulticopterPositionControl::parameters_update(bool force)
 					    "Manual speed has been constrained by maximum speed", _param_mpc_xy_vel_max.get());
 		}
 
+		if (_param_mpc_vel_man_back.get() > _param_mpc_vel_manual.get()) {
+			_param_mpc_vel_man_back.set(_param_mpc_vel_manual.get());
+			_param_mpc_vel_man_back.commit();
+			mavlink_log_critical(&_mavlink_log_pub, "Manual backward speed has been constrained by max speed\t");
```

Comment:
```suggestion
			mavlink_log_critical(&_mavlink_log_pub, "Manual backward speed has been constrained by forward speed\t");
```

Reply:
Should we even do this? Could there be some weird use case where sideways should be allowed faster than forward 🤔 Maybe useless but maybe not worth excluding the possibility.

Reply:
Allowing a larger backward or sideways velocity requires additional changes in the code and I don't think it's worth implementing something more complicated for a potentially nonexistent use-case. I'll change `max` -> `forward` however.